README.md

```diff
@@ -89,112 +89,27 @@ directory (such as Jupyter Lab notebooks) will persist.
-
-# Step 1.1: Create an Amazon SageMaker Image.
-aws sagemaker create-image \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --role-arn $SAGEMAKER_STUDIO_IAM_ROLE_ARN
-
-# Step 2: Create an Amazon SageMaker Image Version.
-aws sagemaker create-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --base-image $ECR_IMAGE_URI
-
-# Optional, step 2.1: Verify that the Image and Image Version were successfully created.
-# You may have to specify a different `--version-number` if you created multiple Image Versions.
-aws sagemaker describe-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --version-number 1
-
-# Step 3.1: Create a file that will contain App Image Config parameters.
-cat > /tmp/app-config.json << EOF
-{
-   "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-   "KernelGatewayImageConfig": { 
-      "FileSystemConfig": { 
-         "DefaultGid": 100,
-         "DefaultUid": 1000,
-         "MountPath": "/home/sagemaker-user"
-      },
-      "KernelSpecs": [ 
-         { 
-            "DisplayName": "Python 3 (ipykernel)",
-            "Name": "python3"
-         }
-      ]
-   }
-}
-EOF
-
-# Step 3.2: Create an Amazon SageMaker App Image Config.
-aws sagemaker create-app-image-config \
-    --cli-input-json file:///tmp/app-config.json
-
-# Step 4.1: Create a file that will contain default parameters for User Settings.
-cat > /tmp/default-user-settings.json << EOF
-{
-    "DefaultUserSettings": {
-        "KernelGatewayAppSettings": {
-            "CustomImages": [
-                {
-                    "ImageName": "$SAGEMAKER_IMAGE_NAME",
-                    "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-                    "ImageVersionNumber": 1
-                }
-            ]
-        }
-    }
-}
-EOF
-
-# Step 4.2: Update Amazon SageMaker Domain with the new default User Settings.
-aws sagemaker update-domain \
-    --domain-id $SAGEMAKER_STUDIO_DOMAIN_ID \
-    --cli-input-json file:///tmp/default-user-settings.json
-```
-
-Now, for your User Profile, log-out and log back in to start using the custom image.
+To use the sagemaker-distribution image in SageMaker Studio, simply go to the image selection dropdown, and select `SageMaker Distribution v{Major_version} {CPU/GPU}`.
 
 ### "I want to directly use the Conda environment, not via a Docker image"
 
 Amazon SageMaker Distribution supports full reproducibility of Conda environments, so you don't necessarily need to use
 Docker. Just find the version number you want to use in the [build_artifacts](build_artifacts) directory, open one of
 _cpu.env.out_ or _gpu.env.out_ and follow the instructions in the first 2 lines.
 
+## Customizing image
+
+When installing packages onto the image, best practice would be to use `micromamba install --no-update-deps --channel conda-forge foo` rather than `pip install`, in order to prevent breaking the environment, while also avoiding unwanted changes to dependencies.
```

Comment:
it is "--freeze-installed" and not "--no-update-deps"

Reply:
Nit: Instead of dependencies, should we specify something like pre-installed etc ? Users might not understand what do we mean with dependencies.

Reply:
Please reword to:

>  Customize the image

> If you'd like to create a new Docker image on top of what we offer, we recommend you use `micromamba install ...` instead of `pip install ...`.

Also, can we give a full working example that we've tested at our end? For e.g., we could install `vim` through `apt-get` and some Python package.

@balajisankar15 - I think both `--freeze-installed` and `--no-update-deps` achieve the same thing. (I am fine either way though - but just check the documentation.)

Reply:
I think --no-update-deps is a conda flag and not a micromamba flag. @claytonparnell  can you check this before making the change.

Reply:
@ketan-vijayvargiya so I should not include explicit recommendation to use --freeze-installed as well? And should I add recommendation elsewhere to use micromamba for when users are just installing packages while using the image? Or we're only recommending this for building a new image?